src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIOptions.cs

```diff
@@ -262,6 +262,7 @@ public class InterceptorFunctions
         /// Accepts one argument requestInterceptor(request) and must return the modified request, or a Promise that resolves to the modified request.
         /// Ex: "function (req) { req.headers['MyCustomHeader'] = 'CustomValue'; return req; }"
         /// </summary>
+        [JsonPropertyName("RequestInterceptorFunction")]
```

Comment:
Given that the rest of the config objects are serialized to camel case, I wonder would it be better to maintain that consistency and just update the assignment in `index.html`?

https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/432c417ee5f7d1598de837495e4de97853e8bb4d/src/Swashbuckle.AspNetCore.SwaggerUI/index.html#L99-L103

Reply:
Just a thought but not a showstopper for me other than that looks good 👍

Reply:
doh didn't realize we had auto-merge - oh well I guess we leave it as is